src/components/header/Header.js

```diff
@@ -14,7 +14,6 @@ const Header = ({findMovie, renderMainPage}) => {
           name= 'searchInput'
```

Comment:
Don't forget that any input should almost always be it's own component, so that it can be a controlled component.